src/natcap/invest/ui/forest_carbon.py

```diff
@@ -11,7 +11,7 @@ def __init__(self):
             label='Forest Carbon Edge Effect Model',
             target=natcap.invest.forest_carbon_edge_effect.execute,
             validator=natcap.invest.forest_carbon_edge_effect.validate,
-            localdoc='forest_carbon_edge_effect.html')
+            localdoc=natcap.invest.forest_carbon_edge_effect.ARGS_SPEC.userguide_html)
```

Comment:
This is very smart... I don't think all the models do this to reduce code redundancy.

Reply:
I think none of them do it because we decided not to overhaul all the `ui/` source code that's destined to be obsolete. But a lot of the data in these files are now in `ARGS_SPEC`.